commit-msg.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 COMMIT_MESSAGE=`head -n1 $1`
-PATTERN="^(PRO|PRI)-[0-9]+: "
+PATTERN="^PRO-[0-9]+: |^PRI-[0-9]+: "
```

Comment:
I think we'd be better to use the original regex pattern as its clearer and has less duplication.